aws/resource_aws_dax_cluster.go

```diff
@@ -95,6 +96,21 @@ func resourceAwsDaxCluster() *schema.Resource {
 				Elem:     &schema.Schema{Type: schema.TypeString},
 				Set:      schema.HashString,
 			},
+			"server_side_encryption": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
```

Comment:
I was able to remove the following by changing the schema definition:
* `Computed: true`
* `CustomizeDiff`
* `isDaxClusterOptionDisabled`

Check this out:

```go
"server_side_encryption": {
	Type:     schema.TypeList,
	Optional: true,
	MaxItems: 1,
	DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
		if old == "1" && new == "0" {
			return true
		}
		return false
	},
	Elem: &schema.Resource{
		Schema: map[string]*schema.Schema{
			"enabled": {
				Type:     schema.TypeBool,
				Optional: true,
				Default:  false,
			},
		},
	},
},
```

At some point I need to make that `DiffSuppressFunc` into its own function 😅 , its basically just there to ignore the container attribute being unset in the Terraform configuration (all child attribute differences still apply).

Reply:
👍 I reused the logic from [`aws_dynamodb_table`](https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/resource_aws_dynamodb_table.go#L43-L51) so we may want to refactor there also.

Reply:
Especially as it looks like with [AWS SDK v1.15.16](https://github.com/aws/aws-sdk-go/releases/tag/v1.15.16) `aws_dynamodb_table` SSE can be updated without recreating the table.